lib/Remit.js

```diff
@@ -24,14 +26,24 @@ class Remit {
     this._options = {}
 
     this._options.exchange = options.exchange || 'remit'
-    this._options.name = options.name || process.env.REMIT_NAME || ''
+    this._options.name = options.name || process.env.REMIT_NAME || 'remit'
     this._options.url = options.url || process.env.REMIT_URL || 'amqp://localhost'
 
     this._emitter = new EventEmitter()
     this._connection = this._connect(this._options).catch(throwAsException)
     this._workers = ChannelPool(this._connection)
     this._publishChannels = {}
 
+    this._tracer = options.tracer || new Tracer({
+      serviceName: this._options.name,
+      reporter: {
```

Comment:
Should reporter be configurable?

Reply:
It might come handy to be able to set the hostname, and port if possible, of the tracer.

Reply:
So the default tracer from OpenTracing is a no-op one, meaning it does bugger all and meaning you don't need to be able to configure it.

If you want to use your own (which you _will_ to do any tracing), you pass in the entire tracer as the `tracer` option (hence `options.tracer || new Tracer()`). With that, you can use your own tracer and reporter config (etc). Example [here](https://github.com/jpwilliams/remit/pull/84/files/6bf181b01df3a507cbf3a1afeca1ef180574791c#diff-b00bc66e819e0ac050c0a67f8474277bR19).